### PR TITLE
Add ApiVersion and Description support to webhook endpoints

### PR DESCRIPTION
**Pull request by APErebus**
As part of this, updates the stripe go dependency to v73, renames the module to `OctopusDeploy/terraform-provider-stripe` and adds a github action to build the artifacts

### CODE REVIEW

File: stripe/resource_stripe_webhook_endpoint.go [hunk 1]
```diff
@@ -23,6 +22,15 @@ func resourceStripeWebhookEndpoint() *schema.Resource {
...
 			},
+			"api_version": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```
Comment (APErebus) [inline stripe/resource_stripe_webhook_endpoint.go:33]: Define the 2 new fields in the schema

File: stripe/resource_stripe_webhook_endpoint.go [hunk 2]
```diff
@@ -54,6 +64,14 @@ func resourceStripeWebhookEndpointCreate(d *schema.ResourceData, m interface{})
 		params.Connect = stripe.Bool(connect.(bool))
 	}
 
+	if apiVersion != "" {
+		params.APIVersion = stripe.String(apiVersion)
+	}
+
+	if description != "" {
+		params.Description = stripe.String(description)
+	}
```
Comment (APErebus) [inline stripe/resource_stripe_webhook_endpoint.go:73]: They are both optional, so don't add them if not provided

File: stripe/resource_stripe_webhook_endpoint.go [hunk 3]
```diff
@@ -96,6 +116,10 @@ func resourceStripeWebhookEndpointUpdate(d *schema.ResourceData, m interface{})
 		params.Connect = stripe.Bool(d.Get("connect").(bool))
 	}
 
+	if d.HasChange("description") {
+		params.Description = stripe.String(d.Get("description").(string))
+	}
```
Comment (APErebus) [inline stripe/resource_stripe_webhook_endpoint.go:121]: The description can be updated

File: stripe/resource_stripe_webhook_endpoint.go [hunk 4]
```diff
@@ -76,6 +94,8 @@ func resourceStripeWebhookEndpointRead(d *schema.ResourceData, m interface{}) er
 	d.Set("url", webhookEndpoint.URL)
 	d.Set("enabled_events", webhookEndpoint.EnabledEvents)
 	d.Set("connect", webhookEndpoint.Application != "")
+	d.Set("api_version", webhookEndpoint.APIVersion)
+	d.Set("description", webhookEndpoint.Description)
```
Comment (APErebus) [inline stripe/resource_stripe_webhook_endpoint.go:98]: Read them into the object

**Review by tleed5 — APPROVED** [review]

**Pull request merged**